plugins/jokes.py

```diff
@@ -1,95 +1,98 @@
-import codecs
-import os
 import random
+from pathlib import Path
 
 from cloudbot import hook
 
+yo_momma = do_it = puns = confucious_say = one_liner = wise_quotes = book_puns = lawyer_jokes = kero_sayings = ['']
```

Comment:
It may be cleaner to just use a single global `dict` indexed by the joke type

Reply:
Yes, yes